src/main/java/org/prebid/server/bidder/epsilon/EpsilonBidder.java

```diff
@@ -59,10 +64,14 @@ public class EpsilonBidder implements Bidder<BidRequest> {
     private final boolean generateBidId;
     private final JacksonMapper mapper;
 
-    public EpsilonBidder(String endpointUrl, boolean generateBidId, JacksonMapper mapper) {
+    public EpsilonBidder(String endpointUrl,
+                         boolean generateBidId,
+                         JacksonMapper mapper,
+                         CurrencyConversionService currencyConversionService) {
         this.endpointUrl = HttpUtil.validateUrl(Objects.requireNonNull(endpointUrl));
         this.generateBidId = generateBidId;
         this.mapper = Objects.requireNonNull(mapper);
+        this.currencyConversionService = Objects.requireNonNull(currencyConversionService);
```

Comment:
Is `currencyConversionService` always available?  Otherwise our bidder could not participate in the auction even if it's all USD?

Reply:
Always available.

Reply:
yes